src/io/flutter/inspector/HeapState.java

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Did you confirm this is the correct license? Elsewhere we use the Chromium license text.

Reply:
Switching to Chromium license since all ours is that - even though Android Studio is different.